src/math.rs

```diff
@@ -78,7 +201,7 @@ pub struct OrientedBeam {
     translation_inv: Vector3<f64>,
     half_extent: Vector2<f64>,
     corners: [Point3<f64>; 4],
-    separating_axes: [Vector3<f64>; 5],
+    separating_axes: Vec<Vector3<f64>>,
```

Comment:
What's the benefit of a vector?

Reply:
Ah, you're filtering before intersects().

Reply:
Yes, so we can avoid the check later.